### ISSUE TITLE
Unhandled rejection TypeError: Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by DennisAhaus** [tjunnone/npm-check-updates#128]
Just executed following console command:

```sh
ncu -g
```

and got the following error:

```js
Unhandled rejection TypeError: Cannot read property 'indexOf' of undefined
    at c:\dev\iojs\node_modules\npm-check-updates\lib\versionmanager.js:383:23
    at baseFindIndex (c:\dev\iojs\node_modules\npm-check-updates\node_modules\lodash\index.js:326:11
)
    at Function.<anonymous> (c:\dev\iojs\node_modules\npm-check-updates\node_modules\lodash\index.js
:3241:23)
    at c:\dev\iojs\node_modules\npm-check-updates\lib\versionmanager.js:382:18
    at Function.<anonymous> (c:\dev\iojs\node_modules\npm-check-updates\node_modules\lodash\index.js
:2984:35)
    at getPreferredWildcard (c:\dev\iojs\node_modules\npm-check-updates\lib\versionmanager.js:381:20
)
    at Object.upgradeDependencies (c:\dev\iojs\node_modules\npm-check-updates\lib\versionmanager.js:
127:20)
    at c:\dev\iojs\node_modules\npm-check-updates\lib\npm-check-updates.js:70:39
    at tryCatcher (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bluebird\js\main\util.js:
26:23)
    at Promise._settlePromiseFromHandler (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bl
uebird\js\main\promise.js:503:31)
    at Promise._settlePromiseAt (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bluebird\js
\main\promise.js:577:18)
    at Promise._settlePromises (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bluebird\js\
main\promise.js:693:14)
    at Async._drainQueue (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bluebird\js\main\a
sync.js:123:16)
    at Async._drainQueues (c:\dev\iojs\node_modules\npm-check-updates\node_modules\bluebird\js\main\
async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (c:\dev\iojs\node_modules\npm-check-updates\nod
e_modules\bluebird\js\main\async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:371:17)
```

**Comment by LunarArcanus** [comment]
Are you sure you have any packages installed globally? Try doing it without the -g flag. @DennisAhaus

**Comment by metaraine** [comment]
Thanks! Definitely a bug.

Fixed in v2.2.1.

**Comment by DennisAhaus** [comment]
Great! Thanks
